core/src/components/range/test/states/range.e2e.ts

```diff
@@ -72,3 +72,27 @@ configs({ directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
     });
   });
 });
+
+configs({ modes: ['ionic-md'], directions: ['ltr'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('range: focus'), () => {
+    test('should render focus state', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-range>
+          <ion-icon name="volume-off" slot="start"></ion-icon>
+          <ion-icon name="volume-high" slot="end"></ion-icon>
+        </ion-range>
+      `,
+        config
+      );
+
+      const range = page.locator('ion-range');
+
+      const handle = await range.locator('.range-knob-handle');
+
+      handle?.focus();
```

Comment:
```suggestion
      handle.focus();
```
The optional chaining isn't needed since we know that the element exists.